cmd/registry/metrics/metrics.go

```diff
@@ -0,0 +1,51 @@
+package metrics
+
+import (
+	"github.com/apigee/registry/rpc"
+)
+
+// ComputeMetrics will compute the metrics proto for a list of Classified Diffs.
+func ComputeMetrics(calculatedDiffs []*rpc.ClassifiedChanges) *rpc.Metrics {
+	stats := calculateStats(calculatedDiffs)
+	return caluclateMetrics(stats)
+}
+
+func calculateStats(calculatedDiffs []*rpc.ClassifiedChanges) *rpc.Stats {
+	totalBreakingChanges := 0
+	totalNonbreakingChanges := 0
+	for _, caluclatedDiff := range calculatedDiffs {
+
+		totalBreakingChanges = (totalBreakingChanges +
+			len(caluclatedDiff.BreakingChanges.Additions) +
+			len(caluclatedDiff.BreakingChanges.Deletions) +
+			len(caluclatedDiff.BreakingChanges.Modifications))
```

Comment:
Nit: Consider adding incrementally instead of computing and assigning the whole sum.

```
totalBreakingChanges += len(...Additions)
totalBreakingChanges += len(...Deletions)
totalBreakingChanges += len(...Modifications)
```

Reply:
Done